sites/svelte.dev/content/blog/2023-01-01-whats-new-in-svelte-january-2023.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "What's new in Svelte: January 2022"
+description: "SvelteKit 1.0 release, directions in custom transitions and new HTML/Svelte typings"
```

Comment:
directions in custom transitions doesn't feel as big as the other two to me. I don't think we have to have three
```suggestion
description: "SvelteKit 1.0 release and type definitions for Svelte elements."
```